src/components/security-checks/styles.ts

```diff
@@ -0,0 +1,48 @@
+import { StyleSheet, Dimensions } from 'react-native';
+import { ITheme } from '../../core/theme/itheme';
+import { BASE_DIMENSION, normalize } from '../../styles/dimensions';
+
+export default (theme: ITheme) =>
+    StyleSheet.create({
+        container: {
+            flex: 1,
+            paddingBottom: BASE_DIMENSION * 2,
+            backgroundColor: theme.colors.appBackground
+        },
+        content: {
+            flex: 1,
+            paddingHorizontal: BASE_DIMENSION * 3,
+            paddingTop: BASE_DIMENSION * 2
+        },
+        bottomButton: {
+            width: '100%'
+        },
+        confirmTextContainer: {
+            flexDirection: 'row',
+            marginBottom: BASE_DIMENSION * 2
+        },
+        textStyle: {
+            color: theme.colors.text,
+            textAlign: 'center',
+            paddingHorizontal: BASE_DIMENSION * 6
+        },
+        title: {
+            fontSize: normalize(25),
+            lineHeight: normalize(30)
+        },
+        message: {
+            fontSize: normalize(18),
+            lineHeight: normalize(25),
+            marginTop: BASE_DIMENSION * 6
+        },
+        imageContainerStyle: {
+            flex: 1,
+            alignItems: 'center',
+            justifyContent: 'center'
+        },
+        imageStyle: {
+            width: Dimensions.get('window').width * 0.5,
```

Comment:
We also have `pw` and `ph` for this
![image](https://user-images.githubusercontent.com/32679726/78682070-36820500-78f6-11ea-8eac-d19bee2b675d.png)

Reply:
uh nice! :) i didn't knew about them